spec.md

```diff
@@ -100,6 +110,18 @@ interface ExpressionStatement <: Statement {
 
 An expression statement, i.e., a statement consisting of a single expression.
 
+## Directive
+
+```js
+interface Directive <: ExpressionStatement {
+    directive: string;
+}
+```
+
+A directive from the directive prologue of a script or function.
+The `directive` property is the raw string source of the directive.
+This source includes single or double wrapping quotes and may not contain an EscapeSequence or LineContinuation..
```

Comment:
In fact, @sebmck is right - making parent to know anything about it's children is really, really bad even if it's just a boolean flag which depends on type of child node. Instead, we chould solve CST in general and provide way to distinguish parenthesized expressions from regular ones.